go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/gorilla/websocket v1.4.0
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/norman v0.0.0-20191118171605-5a9f64e84e65
+	github.com/prometheus/client_golang v1.4.0
```

Comment:
Can you put this to 	github.com/prometheus/client_golang v1.0.0 to match what is in norman?

Reply:
Norman is getting bumped to v1.4.0